# Apply this patch to your Erlang/OTP sources in order to
# chain function calls like
#
#   Person:new(id, "Joe"):save()
#
# To apply it, cd into your OTP source directory and type
#
#   patch -p1 < /path/to/this/patch
#
--- a/lib/stdlib/src/erl_parse.yrl	2013-02-25 13:21:31.000000000 -0600
+++ b/lib/stdlib/src/erl_parse.yrl	2013-03-26 18:54:55.000000000 -0500
@@ -253,7 +253,7 @@
 expr_700 -> record_expr : '$1'.
 expr_700 -> expr_800 : '$1'.
 
-expr_800 -> expr_max ':' expr_max :
+expr_800 -> expr_700 ':' expr_max :
 	{remote,?line('$2'),'$1','$3'}.
 expr_800 -> expr_max : '$1'.
 
